tests/jeos/firstrun.pm

```diff
@@ -405,6 +405,16 @@ sub run {
     verify_norepos unless is_opensuse;
     verify_bsc if is_jeos;
     verify_partition_label;
+
+    if (has_selinux_by_default) {
+        validate_script_output("sestatus", sub { m/SELinux status: .*enabled/ });
+    } else {
```

Comment:
You likely need to limit this to distributions, where SELinux is available. This means skip this check on SLES<16 and Leap<16.

Reply:
It does `which sestatus` so I guess it should work everywhere?

Reply:
Perhaps a new helper "has_selinux" should be created?

Reply:
`has_selinux` would be wrong here because some test might enable SELinux after firstrun